etc/sequencer_solo.out

```diff
@@ -0,0 +1,4 @@
+TESTING writeback_success_state with pubber opts: noWriteback
```

Comment:
I would turn this into an in/out file, that is also the input to the testing, e.g.:
```
TEST writeback_success_state noWriteback RESULT fail writeback_success_state timeout waiting for point filter_differential_pressure_setpoint to have value_state applied
```
as one line could automatically be parsed to extract the first part so it's not replicated in the test script

Reply:
Changed as requested